roles/pre_tasks/tasks/main.yml

```diff
@@ -1,10 +1,23 @@
 ---
 
 # Install absolutely necessary packages
+
+- name: Check if the absolutely necessary packages are installed
+  shell: "type {{ item }}"
+  with_items: "{{ tower_pre_tasks_packages }}"
+  register: type_packages
```

Comment:
A bit unfair I know but while I'm at it: following the [best practices from oasis](https://github.com/oasis-roles/meta_standards/blob/master/README.md), the variable should be called something like `__towerutils_check_packages`.

Reply:
Updated, Agree with changes